linux/uwecho.bsh

```diff
@@ -36,50 +36,81 @@ source "${VSI_COMMON_DIR}/linux/compat.bsh"
 #
 # .. code-block:: bash
 #
-#   $   echo "This
-#   > is
-#   >   a
-#   >   test"
+#    $   echo "This
+#    > is
+#    >   a
+#    >   test"
 #
 # uwecho will determine how many spaces up to the quote on the first line, and remove that many spaces if they are leading in all the subsequent lines
 #
 #
 # .. code-block:: bash
 #
-#   $   uwecho "This
-#   >           is
-#   >             a
-#   >             test"
+#    $   uwecho "This
+#    >           is
+#    >             a
+#    >             test"
 #
 # Much easier to read. Even handles:
 #
 # .. code-block:: bash
 #
-#   $   uwecho "  This
-#   >           is
-#   >             a test"
+#    $   uwecho "  This
+#    >           is
+#    >             a test"
 #
 # One odd side-effect is this looks a little weird:
 #
 # .. code-block:: bash
 #
-#  $   uwecho 'Use "quote" and get: '"${Variable}"'
-#              '"${Another_variable}"' goes here
-#              * <-- This is where the indent is'
+#    $   uwecho 'Use "quote" and get: '"${Variable}"'
+#                '"${Another_variable}"' goes here
+#                * <-- This is where the indent is'
 #
 # The second line looks like it has one too many spaces when, in fact, it has the right number of spaces.
 #
 # .. note::
-#   - This only works in when called directly from a script saved in a file. Will not work in a script that is piped in or on interactive command line.
-#   - Will not work correctly if you are using ``\n``
-#   - Will not work correctly if you use hard tab in your indents. Use here doc if you want to use hard tabs
-#   - Not smart enough to work unless line starts with spaces and ``uwecho``. No inline environment variables setting, or execution of wrappers, etc...
-#   - Does not work if you cd after the script is started, unless you use full path name to call the script
-#   - Does not work in $() or ` ` or any as any subprocess
+#
+#    - This only works in when called directly from a script saved in a file. Will not work in a script that is piped in or on interactive command line.
+#    - Will not work correctly if you are using ``\n``
+#    - Will not work correctly if you use hard tab in your indents. Use here doc if you want to use hard tabs
+#    - Not smart enough to work unless line starts with spaces and ``uwecho``. No inline environment variables setting, or execution of wrappers, etc...
+#    - Does not work if you cd after the script is started, unless you use full path name to call the script
+#    - Does not work in $() or ` ` or any process subprocess
```

Comment:
This is quite a list. I think I understand what most of these limitations mean. But do you think an example of some (each) would be worth it?

Reply:
yes